app/src/androidTest/java/com/appunite/loudius/util/IntegrationTestRule.kt

```diff
@@ -18,36 +18,27 @@ package com.appunite.loudius.util
 
 import androidx.activity.ComponentActivity
 import androidx.compose.ui.test.junit4.createAndroidComposeRule
-import com.appunite.loudius.TestActivity
+import androidx.compose.ui.test.junit4.createComposeRule
 import com.appunite.loudius.components.components.countingResource
 import com.appunite.loudius.util.IdlingResourceExtensions.toIdlingResource
-import dagger.hilt.android.testing.HiltAndroidRule
+import net.bytebuddy.dynamic.TypeResolutionStrategy.Active
```

Comment:
Question: What is this for??🤓

Reply:
no idea O.o I forgot to delete the imports that were unused ;s